File: leetcode/1480. Running Sum of 1d Array/soln.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        running_sum = []
        curr_sum = 0
        for el in nums:
            curr_sum += el
            running_sum.append(curr_sum)
        return running_sum

        

File: leetcode/problems/1480. Running Sum of 1d Array/soln.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        int sum{0U};
        vector<int> run_sum;
        for (auto i:nums){
            sum += i;
            run_sum.push_back(sum);
        }
        return run_sum;
        
    }
};